backend/app.py

```diff
@@ -751,6 +761,43 @@ def sendEmail():
         return jsonify({"error": str(e)}), 500
 
 
+@app.route("/api/createInvitation", methods=["POST"])
```

Comment:
@Alexjes98 the name of the endpoint shouldn't be just POST **/api/invitations** instead of **/api/createInvitation**?

https://restfulapi.net/resource-naming/

Reply:
I don't get it you mean the sendEmail endpoint?